src/operator/nn/fully_connected-inl.h

```diff
@@ -47,7 +48,24 @@ namespace fullc {
 enum FullyConnectedOpInputs {kData, kWeight, kBias};
 enum FullyConnectedOpResource {kTempSpace};
 enum FullyConnectedOpOutputs {kOut};
-}  // fullc
+enum FullyConnectedGradGradOutputs {
+  kOyGrad,
+  kXGradGrad,
+  kWGradGrad,
+  kBGradGrad
+};
+enum Inputs {
```

Comment:
A more specific name?

Reply:
Please suggest better ones in these circumstances. It would help, this is the best I could came up with and I'm not a native speaker.